pkg/cmd/certregenerationcontroller/cabundlesyncer.go

```diff
@@ -0,0 +1,149 @@
+package certregenerationcontroller
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	corev1listers "k8s.io/client-go/listers/core/v1"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+	"k8s.io/klog"
+
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/operatorclient"
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/targetconfigcontroller"
+)
+
+const workQueueKey = "key"
+
+type CABundleController struct {
+	kubeClient kubernetes.Interface
```

Comment:
you only need `ConfigMapsGetter` right?

Reply:
I think the applyConfig map required live client too

Reply:
done